package.json

```diff
@@ -35,49 +35,50 @@
     "icons:update": "node ./packages/icons/update-icons.js"
   },
   "dependencies": {
-    "@angular/animations": "^16.1.3",
-    "@angular/cdk": "^16.1.3",
-    "@angular/common": "^16.1.3",
-    "@angular/compiler": "^16.1.3",
-    "@angular/core": "^16.1.3",
-    "@angular/forms": "^16.1.3",
-    "@angular/platform-browser": "^16.1.3",
-    "@angular/platform-browser-dynamic": "^16.1.3",
-    "@angular/router": "^16.1.3",
+    "@angular/animations": "^17.0.2",
+    "@angular/cdk": "^17.0.0",
+    "@angular/common": "^17.0.2",
```

Comment:
We must reflect these bumps in the ng/package.json file to allow apps to update to Angular 17.
(GitHub won't let me comment on the whole file, sorry)